src/main/java/com/GeorgesServer/app/App.java

```diff
@@ -1,16 +1,24 @@
 package com.GeorgesServer.app;
 
-import java.io.*;
 
 public class App {
-    public static void main(String args[]) throws IOException {
+    public static void main(String args[]) {
+
+        if (args.length != 4) {
```

Comment:
This is like having a debug statement you forgot to get rid of before you committed.